docs/labs/13-Refact.md

```diff
@@ -575,11 +575,11 @@ Credit will be given to how much more readable,
 flexible and possibly simple the code is in relation to the original.
 **You can work on this project in pairs,
 but should not break up or attempt to break up the work.
-Rather you should:
+Rather you should:**
 
-1. work on it together,
-2. propose ideas to each other about how to refactor
-3. propose ideas on what should be refactored.**
+**1. work on it together,**<br>
+**2. propose ideas to each other about how to refactor,**<br>
```

Comment:
Are the `<br>` tags necessary?  I don't think they are necessary.
I'm going to have you remove them from here.

Reply:
Just removed it